src/MAUI/Maui.Samples/Samples/Layers/AddVectorTiledLayerFromCustomStyle/readme.md

```diff
@@ -0,0 +1,37 @@
+# Add vector tiled layer from custom style
+
+Load ArcGIS vector tiled layers using custom styles.
+
+![Image of offline vector tiled layer custom style](AddVectorTiledLayerFromCustomStyle.jpg)
+
+## Use case
+
+Vector tile basemaps can be created in ArcGIS Pro and published as offline packages or online services. You can create a custom style tailored to your needs and easily apply them to your map. `VectorTiledLayer` has many advantages over traditional raster based basemaps (`ArcGISTiledLayer`), including smooth scaling between different screen DPIs, smaller package sizes, and the ability to rotate symbols and labels dynamically.
```

Comment:
Could be helpful to include some context around the use of custom styles specifically. Personally, I'd remove the last sentence under "Use Case" and include some of the information provided in the relevant documentation.

https://doc.arcgis.com/en/arcgis-online/reference/update-vector-tile-style.htm